core/cont/src/TList.cxx

```diff
@@ -714,7 +714,7 @@ void TList::RecursiveRemove(TObject *obj)
    while (lnk) {
       next = lnk->Next();
       TObject *ob = lnk->GetObject();
-      if (ob->TestBit(kNotDeleted)) {
+      if (ob && ob->TestBit(kNotDeleted)) {
```

Comment:
@pcanal, note that using this coding pattern you avoid such nullptr derefs at a slight indentation cost. I would not support choosing indentation over correctness.